.github/workflows/build.yml

```diff
@@ -298,6 +298,26 @@ jobs:
           MB_VERSION=`./target/release/moonbeam --version`
           echo "Checking $MB_VERSION contains $GIT_COMMIT"
           echo "$MB_VERSION" | grep $GIT_COMMIT
+      - name: Save runtime wasm
```

Comment:
Why change the order ?

Reply:
because running `cargo test` actually changes the RT that gets created in `/wbuild`!

The resulting blob is incompatible with uploading and the chain cannot be successfully upgraded using it.